frontend/src/stores/configuration.ts

```diff
@@ -12,6 +13,19 @@ export const useConfiguration = defineStore('configuration', {
     },
   },
   getters: {
+    getTokenForChain:
+      (state) =>
+      (chainId: number, tokenSymbol: string): Token | undefined => {
+        const chain = state.chains[chainId];
+        if (!chain) return;
+
+        return chain.tokens.find((token) => token.symbol === tokenSymbol);
+      },
+    getChain:
```

Comment:
Currently, it's not. 
I remember i wrote this function while refactoring the components (https://github.com/beamer-bridge/beamer/pull/918/files#diff-8d9de3a4223dc705759f7cff855b2e7400e353863878915795fc96f4426b8efcL107-L113) and in the end didn't really check if its still used somewhere. 
I can remove it and the tests related to it, but i can already see that this simple getter could be used in the future.
What do you think?

Reply:
In general, I think we should not have code in the repository, just because it might get used but we are not sure about it. In this case, the getter is so small that I would just leave it.